brianlong/www.validators.app
# 187304004 update rubocop config

#### What's this PR do?
- updates rubocop config and upgrades gems

#### How should this be manually tested?
- bundle install
- run `bundle exec rubocop`
- if you see errors related to loading rubocop-rails file, run `rubocop --restart-server` first, and then rubocop check again
- confirm it runs without errors

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/187304004)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
